src/signHash.c

```diff
@@ -110,7 +115,7 @@ void handleSignHash(
 #ifdef HAVE_BAGL
     ux_flow_init(0, ux_approve_hash_flow, NULL);
 #else
-    snprintf(ctx->typeStr, sizeof(ctx->typeStr), "Sign hash with key %d?", ctx->keyIndex);
+    snprintf(ctx->typeStr, sizeof(ctx->typeStr), "Sign Hash with Key %d?", ctx->keyIndex);
     nbgl_useCaseReviewStart(&C_stax_app_sia,
```

Comment:
This doesn't follow the guidelines.
And for blind signing you should only expose it if user as enabled a settings which clearly mentions the risk.